openmc/mgxs/library.py

```diff
@@ -773,9 +774,22 @@ def get_xsdata(self, domain, xsdata_name, nuclide='total', xs_type='macro',
             nuclide this will be set to 'macro' regardless.
         xs_ids : str
             Cross section set identifier. Defaults to '1m'.
-        order : Scattering order for this data entry.  Default is None,
+        order : int
+            Scattering order for this data entry.  Default is None,
             which will set the XSdata object to use the order of the
             Library.
+        tabular_legendre : {None, bool}
+            Flag to denote whether or not the Legendre expansion of the
+            scattering angular distribution is to be converted to a tabular
+            representation by OpenMC.  A value of `True` means that it is to be
+            converted while a value of 'False' means that it will not be.
+            Defaults to `None` which leaves the default behavior of OpenMC in
+            place (the distribution is converted to a tabular representation).
+        tabular_points : {int}
+            This parameter is not used unless the `tabular_legendre` is set to
+            `True`.  In this case, this parameter sets the number of
```

Comment:
If you want fixed-width text, you need to use double back-ticks instead of single, ``` ``tabular_legendre`` ```